R/timer.R

```diff
@@ -42,6 +42,17 @@ TimerCallbacks <- R6Class(
 
       return(id)
     },
+    unschedule = function(id) {
+      toRemoveIndices <- .times$id %in% id
+      toRemoveIds <- .times[toRemoveIndices, "id", drop = TRUE]
+      if (length(toRemoveIds) > 0) {
+        .times <<- .times[!toRemoveIndices,]
+        for (toRemoveId in as.character(toRemoveIds)) {
+          .funcs$remove(toRemoveId)
+        }
+      }
+      return(id %in% toRemoveIds)
```

Comment:
This matches the prior if statement.  Maybe a explicit return(TRUE) inside line 53 and return(FALSE) otherwise?

Reply:
It's a vectorized result, so you can see which of the id's you passed in were actually found and removed.

Reply:
Sorry, I guess I didn't make this explicit: you can pass in a vector multiple id's.